app/assets/stylesheets/_search_result.scss

```diff
@@ -4,6 +4,9 @@
     padding: 25px 0;
   }
   &-container {
+  {
```

Comment:
I guess my original comment about the position relative attribute is that I cant see if serving any functional purpose. That is to say, adding this property doesnt actually change the page from what I can see so I dont think its actually needed.